solutions/tf-fullstack-da/README.md

```diff
@@ -0,0 +1,76 @@
+# Terraform based "fullstack" sample deployable architecture - COS bucket replication
+
+This architecture supports provisioning the following resources:
+- Resource group (also support using existing resource group)
+- A global Cloud Object Storage instance
+- An primary source Object Storage bucket in a given region
+- An target Object Storage bucket where data will be replicated to
+- A replication rule to replicate everything from the source bucket to the target bucket
+- An IAM authorization policy required for the replication between the buckets
```

Comment:
```suggestion
# Sample terraform-based deployable architecture without dependencies (fullstack) - COS bucket replication

This architecture has the `install_type` set to `fullstack` in the `ibm_catalog.json`, which means that it does not have any dependencies and it always deploys the end-to-end solution instead of giving the user the option to extend existing resources in the account. This `fullstack` install type should not to be confused with a [deployable architecture stack](https://cloud.ibm.com/docs/secure-enterprise?topic=secure-enterprise-understand-module-da#what-is-da), which is an architecture built of two or more deployable architectures that are often linked with references and deployed in a particular sequence.

This architecture supports provisioning the following resources:
- Resource group (also support using existing resource group)
- A global Cloud Object Storage instance
- A primary source Object Storage bucket in a given region
- An target Object Storage bucket where data will be replicated to
- A replication rule to replicate everything from the source bucket to the target bucket
- An IAM authorization policy required for the replication between the buckets
```

Again @SirSpidey or @ocofaigh if you can help me make this clarification clearer or think it should be somewhere else, let's figure that out together.

Reply:
> which is an architecture built of two or more deployable architectures

^ this might (hopefully) change to also allow deployable modules, but lets leave as is for now until thats finalised.

Reply:
@SirSpidey if your happy with this, I'll commit it